andela/noldor-ah-backend
# #161816772 Reorganise File and Folder Structure

#### What does this PR do?
This PR fixes the messy structure of the repo. All related functions are moved to specific folders while redundant imports are removed.

#### Description of Task to be completed?
- fix file and folder structure
- remove redundant imports
- make repo easier to traverse

#### How should this be manually tested?
1. Clone the repo and checkout to this branch
2. Notice the differences in the organisation structure

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161816772

#### Screenshots (if appropriate)
N/A

#### Questions
N/A
